README.md

```diff
@@ -109,6 +109,15 @@ If everything is successful, you should see something like this:
 ```
 (master)$ Successfully rebased and updated refs/heads/master.
 ```
+### Possible issues with merging
+Safe merging strategy:
```

Comment:
Thanks for the thanks, not trying to block this, it's a great addition. Ping when you've forced pushed the changes and I'll merge them in.